### ISSUE TITLE
[Accessibility: ACC-255.1] Button: No alternative text provided

### ISSUE THREAD

**Issue by divyamalhotra488** [SAP/fundamental-ngx#12576]
#### Is this a bug, enhancement, or feature request?
Bug

#### Describe your proposal.
For all non-text UI elements text alternatives should be provided and this information should be read out by screen reader.
Every semantic piece of non-text content should be available as text.

#### Can you handle that on the application side
No

#### Which versions of Angular and Fundamental Library for Angular are affected? Please, specify the exact version. (If this is a feature request, use current version.)
All

#### If this is a bug, please provide steps for reproducing it; the exact components you are using;
Component used: [Multi Combobox](https://sap.github.io/fundamental-ngx/#/core/multi-combobox)
There is no text alternative provided for all the ‘Delete/Remove’ buttons within all delete-able tokens.
<img width="452" alt="image" src="https://github.com/user-attachments/assets/f57c3ca6-bda9-4c62-8cab-c69bc4a68e1d">

#### Please provide stackblitz example(s).
You can try ngx examples, issue is reproducible there.

#### In case this is Accessibility related topic, did you consult with an accessibility expert? If not, please do so and share their recommendations.
Yes, issue is raised by an accessibility expert.

#### Did you check the documentation and the API?
Yes

#### Did you search for similar issues?
Yes

#### Is there anything else we should know?

**_IMPORTANT: Please refrain from providing links or screenshots of SAP's internal information, as this project is open-source, and its contents are accessible to anyone._**

**Comment by droshev** [comment]
The keyboard support is by focusing on the token and deleting it with delete button.
